src/app/vault/ciphers.component.html

```diff
@@ -7,6 +7,70 @@
     [infiniteScrollDisabled]="!isPaging()"
     (scrolled)="loadMore()"
   >
+    <thead id="sort-ciphers-table">
+      <tr>
+        <th scope="col">
+          <input (click)="selectAll()" type="checkbox" />
+        </th>
+        <th
+          colspan="2"
+          *ngIf="sortedDescending; else elseBlock1"
+          scope="col"
+          class="show-arrow-on-hover user-select-none"
+        >
+          <i
+            (click)="sortCiphers('name')"
+            class="sorting-arrow sort-name-arrow bwi-arrow-down"
+            aria-hidden="true"
+          ></i
+          >{{ "name" | i18n }}
+        </th>
+        <ng-template #elseBlock1>
+          <th colspan="2" scope="col" class="show-arrow-on-hover user-select-none">
+            <i
+              (click)="sortCiphers('name')"
+              class="sorting-arrow sort-name-arrow bwi-arrow-up"
+              aria-hidden="true"
+            ></i
+            >{{ "name" | i18n }}
+          </th>
+        </ng-template>
+        <th
+          scope="col"
+          colspan="2"
+          class="show-arrow-on-hover table-col-sort cipher-sort-table-header"
+        >
+          <div class="input-group m-0 font-weight-bold">
+            <div
+              (click)="sortCiphers(option.value)"
+              *ngIf="sortedDescending; else elseBlock2"
+              class="input-group-prepend no-bg"
+            >
+              <label class="input-group-text bg-transparent border-0" for="sortOptions"
+                ><i class="sorting-arrow bwi bwi-angle-down" aria-hidden="true"></i
+              ></label>
+            </div>
+            <ng-template #elseBlock2>
+              <div (click)="sortCiphers(option.value)" class="input-group-prepend">
+                <label class="input-group-text bg-transparent border-0" for="sortOptions"
+                  ><i class="sorting-arrow bwi bwi-chevron-up" aria-hidden="true"></i
+                ></label>
+              </div>
+            </ng-template>
+            <select
+              #option
```

Comment:
Get rid of this id (`#option`) - see other comment. You should refer to the class property this control is bound to, not the control itself.